### ISSUE TITLE
Prisma is broken in v0.3.0 Docker image 

### ISSUE THREAD

**Issue by jonleopard** [codetheweb/muse#463]
Hey there,

I've been running Muse great for months now. The latest image you pushed is causing some errors. I've attached some logs coming from the docker container below. I read that you migrated over to Primsa, is there a migration guide somewhere? Or should the docker container do all that for me?

Let me know if I can provide any additional info!

Cheers mate,
- Jon

```
yarn run v1.22.15
$ npm run env:set-database-url -- node --experimental-json-modules dist/src/scripts/migrate-and-start.js

> muse@0.3.0 env:set-database-url
> node dist/src/scripts/run-with-database-url.js "node" "--experimental-json-modules" "dist/src/scripts/migrate-and-start.js"

(node:52) ExperimentalWarning: Importing JSON modules is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
/usr/app/node_modules/.prisma/client/index.js:3
    throw new Error(
          ^

Error: @prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.
In case this error is unexpected for you, please report it in https://github.com/prisma/prisma/issues
    at new PrismaClient (/usr/app/node_modules/.prisma/client/index.js:3:11)
    at file:///usr/app/dist/src/utils/db.js:2:23
    at ModuleJob.run (node:internal/modules/esm/module_job:185:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:281:24)
    at async loadESM (node:internal/process/esm_loader:88:5)
    at async handleMainPromise (node:internal/modules/run_main:65:12)
file:///usr/app/node_modules/execa/lib/error.js:59
                error = new Error(message);
                        ^

Error: Command failed with exit code 1: node --experimental-json-modules dist/src/scripts/migrate-and-start.js
    at makeError (file:///usr/app/node_modules/execa/lib/error.js:59:11)
    at handlePromise (file:///usr/app/node_modules/execa/index.js:119:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async file:///usr/app/dist/src/scripts/run-with-database-url.js:6:5 {
  shortMessage: 'Command failed with exit code 1: node --experimental-json-modules dist/src/scripts/migrate-and-start.js',
  command: 'node --experimental-json-modules dist/src/scripts/migrate-and-start.js',
  escapedCommand: 'node --experimental-json-modules "dist/src/scripts/migrate-and-start.js"',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: undefined,
  stderr: undefined,
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
npm notice
npm notice New minor version of npm available! 8.1.0 -> 8.3.0
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v8.3.0>
npm notice Run `npm install -g npm@8.3.0` to update!
npm notice
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
yarn run v1.22.15
$ npm run env:set-database-url -- node --experimental-json-modules dist/src/scripts/migrate-and-start.js

> muse@0.3.0 env:set-database-url
> node dist/src/scripts/run-with-database-url.js "node" "--experimental-json-modules" "dist/src/scripts/migrate-and-start.js"

(node:52) ExperimentalWarning: Importing JSON modules is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
/usr/app/node_modules/.prisma/client/index.js:3
    throw new Error(
          ^

Error: @prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.
In case this error is unexpected for you, please report it in https://github.com/prisma/prisma/issues
    at new PrismaClient (/usr/app/node_modules/.prisma/client/index.js:3:11)
    at file:///usr/app/dist/src/utils/db.js:2:23
    at ModuleJob.run (node:internal/modules/esm/module_job:185:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:281:24)
    at async loadESM (node:internal/process/esm_loader:88:5)
    at async handleMainPromise (node:internal/modules/run_main:65:12)
file:///usr/app/node_modules/execa/lib/error.js:59
                error = new Error(message);
                        ^

Error: Command failed with exit code 1: node --experimental-json-modules dist/src/scripts/migrate-and-start.js
    at makeError (file:///usr/app/node_modules/execa/lib/error.js:59:11)
    at handlePromise (file:///usr/app/node_modules/execa/index.js:119:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async file:///usr/app/dist/src/scripts/run-with-database-url.js:6:5 {
  shortMessage: 'Command failed with exit code 1: node --experimental-json-modules dist/src/scripts/migrate-and-start.js',
  command: 'node --experimental-json-modules dist/src/scripts/migrate-and-start.js',
  escapedCommand: 'node --experimental-json-modules "dist/src/scripts/migrate-and-start.js"',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: undefined,
  stderr: undefined,
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

**Comment by Quinten0508** [comment]
+1, getting the exact same error here. Running ARM 64 bit, Ubuntu server LTS.

**Comment by codetheweb** [comment]
Sorry about that, should be fixed in v0.3.1.

Currently being [built and pushed](https://github.com/codetheweb/muse/runs/4731755501?check_suite_focus=true).

> I read that you migrated over to Primsa, is there a migration guide somewhere? Or should the docker container do all that for me?

Muse should handle all migrations automatically.
